app-admin/gtkdiskfree/gtkdiskfree-2.0.4.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Graphical tool to show free disk space like df"
+HOMEPAGE="https://gitlab.com/mazes_80/gtkdiskfree"
+SRC_URI="https://gitlab.com/mazes_80/${PN}/-/archive/${PV}/${P}.tar.bz2"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="gtk2 old-color-selector"
+
+DEPEND="
+	dev-libs/glib
+	gtk2? ( x11-libs/gtk+:2 )
+	!gtk2? ( x11-libs/gtk+:3 )
```

Comment:
Please sort this list by putting all conditional deps below unconditional ones, 
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Sorting_dependencies

Reply:
Done.